test/simulator/CMakeLists.txt

```diff
@@ -0,0 +1,191 @@
+# Copyright 2018 Shift Cryptosecurity AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+#-----------------------------------------------------------------------------
+# Build bitbox lib to use in tests
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-unused-parameter -Wno-missing-prototypes -Wno-missing-declarations -Wno-implicit-function-declaration -Wno-bad-function-cast")
+
+set(DBB-FILTERED-SOURCES
+  ${DBB-FIRMWARE-SOURCES}
+  ${DBB-FIRMWARE-UI-SOURCES}
+  ${FIRMWARE-U2F-SOURCES}
+  ${DBB-FIRMWARE-USB-SOURCES}
+  )
+
+set(IGNORE_SOURCES
+    "src/screen.c"
+    "src/memory/nvmctrl.c"
+    "src/memory/smarteeprom.c"
+    "src/memory.mpu.c"
+    )
+#f9b3ceaa91e
```

Comment:
remove this comment here and in the unit-test folder, seems like some local note leftover